tests/rkt_image_rm_test.go

```diff
@@ -265,12 +357,15 @@ func getImageId(ctx *testutils.RktRunCtx, name string) (string, error) {
 	return imageID, nil
 }
 
-func removeImage(ctx *testutils.RktRunCtx, image string, shouldWork bool) error {
-	expect := fmt.Sprintf(rmImageReferenced, image)
+func removeImage(ctx *testutils.RktRunCtx, images []string, shouldWork bool) error {
+	cmd := fmt.Sprintf("%s image rm", ctx.Cmd())
+	for _, image := range images {
+		cmd += fmt.Sprintf(" %s", image)
+	}
+	expect := fmt.Sprintf(rmImageReferenced, images[0])
```

Comment:
When simplifying the function, consider changing its prototype to `func removeImage(ctx *testutils.RktRunCtx, images ...string) error`, or, without returning an error, but instead failing inside the function - `func removeImage(t *testing.T, ctx *testutils.RktRunCtx, images ...string)`. With variadic parameters you could have `removeImage(ctx, "image")` or `removeImage(ctx, "image", "sha512-stuff", "something else")`.